tuf/api/metadata.py

```diff
@@ -443,7 +443,9 @@ def verify_delegate(
                 key.verify_signature(delegated_metadata, signed_serializer)
                 signing_keys.add(key.keyid)
             except exceptions.UnsignedMetadataError:
-                logger.info("Key %s failed to verify %s", keyid, delegated_role)
+                logger.warning(
```

Comment:
This should not be a warning, debug is fine: one of the keys just didn't have a valid sig for this metadata. The metadata may still be signed by enough keys

Reply:
Ugh, my bad. Will fix now.